symbulate/random_variables.py

```diff
@@ -46,11 +71,18 @@ def check_same_probSpace(self, other):
             self.probSpace.check_same(other.probSpace)
 
     def apply(self, function):
-        """
+        """Transform a random variable by a function.
+
         Args:
-            function: function to apply to the random variable (e.g., log, sqrt, exp)
+          function: function to apply to the random variable
         
-        Input function is applied to the output results.
+        Examples:
```

Comment:
I agree with Dr. Ross. I don't remember covering lambda functions in any cs classes so this could also be a good introduction to them for new python programmers.